dspace_metadata_import.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+
+########################################################################################
+## Run this script to import metadata from old dspace into new dspace6 database, and  ##
+## copy over the metadata registry files.											  ##
+########################################################################################
+
+
+## Refer to : https://unix.stackexchange.com/questions/191990/shell-script-to-execute-psql-command ##
+## DSpace6 Upgrading: https://wiki.duraspace.org/display/DSDOC6x/Upgrading+DSpace ##
+
+
+#########################################################################
+#                       									 			#
+#	Need to 															#
+#		1) copy over the dumped pgsql file from old dspace, and			#
+#		2) set up the ownership, and									#
+#		3) replace the user to be dspace or something, and 				#
+#		4) comment out the block of adding rule protect_approval_upd	#
+#    	5) comment out the privileges granted to user libacct			#
+#                       												#
+#########################################################################
+
+PATH_TO_REGISTRIES_FROM_OLD_DSPACE=/tmp/registries.tar.gz
```

Comment:
I think these variables should probably be command line arguments to the script. Remember that we want to be able to use this for the commons install (which I expect to live at /srv/commons) and the shareok install (which I expect to live at /srv/shareok), and we don't want to have to hand edit anything to make that happen. 

Alternately, this script could be a template and you could use Ansible to set these variables based on host vars when the script is installed.

Reply:
If we use the script as a template in the ansible role, we need to prepare the dumped data from old DSpace with paths saved as variables in the ansible role. The problem is, where should we store the dumped data so the ansible role can access them?